Wikipedia/Code/SavedPageSpotlightManager.swift

```diff
@@ -3,25 +3,29 @@ import UIKit
 import MobileCoreServices
 import CoreSpotlight
 
-@available(iOS 9.0, *)
 
-public extension CSSearchableItemAttributeSet {
-    
-    public class func attributes(article: MWKArticle) -> CSSearchableItemAttributeSet {
-        
+public extension NSURL {
+    @available(iOS 9.0, *)
+    func searchableItemAttributes() -> CSSearchableItemAttributeSet {
```

Comment:
This should probably only work for Wikipedia Article URLs.
You can check the `wmf_isWikiResource` property of the URL to make sure that is true.